schools.json

```diff
@@ -0,0 +1,20 @@
+{
+  "schools": [
+    {
+      "id": "pt.ipl.isel",
```

Comment:
I think the country code should be included as well, even though it can sometimes be inferred from the school ID, that may be not always true

Reply:
Added country code.